core/block_svg.js

```diff
@@ -523,14 +524,21 @@ Blockly.BlockSvg.prototype.tab = function(start, forward) {
 
 /**
  * Handle a mouse-down on an SVG block.
- * @param {!Event} e Mouse down event.
+ * @param {!Event} e Mouse down event or touch start event.
```

Comment:
Pointer events are another API (the one used by IE and Edge) and I don't know if the eventual code to handle them will be distinct from this code.